db/migrate/20160811030257_create_players.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePlayers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :players do |t|
+      t.string :first_name
+      t.string :last_name
+
+      t.timestamps null: false
```

Comment:
why do we have timestamps for when the player was created? and what does the false mean?